src/components/Chat/Chat.tsx

```diff
@@ -240,10 +258,21 @@ const ChatMessage = ({
         />
       ) : null}
       <Comment.Content>
-        <Comment.Author as="a" className="light">
-          {Boolean(system) && 'System'}
-          {nameMap[id] || id}
-        </Comment.Author>
+        {owner && owner === user?.uid ? (
+          <UserMenu
```

Comment:
instead of wrapping the author component couldn't we just trigger this using an onClick handler?

Reply:
I think the triggering element is supposed to be wrapped like this. But I found a better way to implement it.